src/io/dataset_loader.cpp

```diff
@@ -2,8 +2,8 @@
  * Copyright (c) 2016 Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License. See LICENSE file in the project root for license information.
  */
-#include <LightGBM/dataset_loader.h>
 
+#include <LightGBM/dataset_loader.h>
```

Comment:
Sorry, I probably was not clear enough, it should be
```
* Licensed under the MIT License. See LICENSE file in the project root for license information.
*/
#include <LightGBM/dataset_loader.h>

#include <LightGBM/network.h>
```

In other words, it should not produce any diff.

Reply:
Ah ok, I get it now. Sorry, I didn't read your previous comment carefully enough!